.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug report
+description: Create a report to help us improve
+labels: Bug
+
+body:
+- type: textarea
+  attributes:
+    label: Describe the bug
+    placeholder: A clear and concise description of what the bug is.
```

Comment:
One thing to consider is to use `description` element instead of `placeholder`.
That way the description is displayed above the input text area (instead of inside the area).

Not sure which one is better though.